index.rst

```diff
@@ -6,7 +6,158 @@ Roadmap for Community Alert Filters with the ANTARES Broker
 
    This document guides the Rubin Community Science team’s work on Community Alert Filters for the ANTARES broker. The motivation, scope, and technical considerations for the community filters are described, and the timeline for community input and filter development is defined.
 
-Add content here
-================
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+Overview
+========
+
+Requirements on the Rubin system specify that users will have access to alerts via predefined filters (Appendix A), hereafter referred to as "community filters."
+These community filters are intended to provide a low barrier-to-entry starting point for community members wanting to start with alert science, and as such should cover a range of Rubin alert use cases.
+The Rubin Community Science team (CST) will take input from the community to define, implement, and maintain the community filters.
+The existence of community filters served by the CST do not preclude users from developing their own customized filters.
+The ANTARES alert broker provides the functionality for user-defined filters, and the ANTARES team has agreed to host these community filters, so that Rubin Operations can meet these requirements without constructing an Alert Filtering Service (`DMTN-226 <https://dmtn-226.lsst.io/>`_).
+
```

Comment:
I'd add links to the ANTARES paper and to https://nsf-noirlab.gitlab.io/csdc/antares/antares/